x-pack/plugins/security_solution/common/api/timeline/model/api.ts

```diff
@@ -251,93 +273,26 @@ export type Sort = runtimeTypes.TypeOf<typeof SavedSortRuntimeType>;
  *  Timeline Statuses
  */
 
-export enum TimelineStatus {
-  active = 'active',
-  draft = 'draft',
-  immutable = 'immutable',
-}
-
 export const TimelineStatusLiteralRt = runtimeTypes.union([
-  runtimeTypes.literal(TimelineStatus.active),
-  runtimeTypes.literal(TimelineStatus.draft),
-  runtimeTypes.literal(TimelineStatus.immutable),
+  runtimeTypes.literal(TimelineStatusEnum.active),
+  runtimeTypes.literal(TimelineStatusEnum.draft),
+  runtimeTypes.literal(TimelineStatusEnum.immutable),
 ]);
 
-const TimelineStatusLiteralWithNullRt = unionWithNullType(TimelineStatusLiteralRt);
-
-export type TimelineStatusLiteralWithNull = runtimeTypes.TypeOf<
-  typeof TimelineStatusLiteralWithNullRt
->;
-
-export enum RowRendererId {
-  /** event.kind: signal */
-  alert = 'alert',
-  /** endpoint alerts (created on the endpoint) */
-  alerts = 'alerts',
-  auditd = 'auditd',
-  auditd_file = 'auditd_file',
-  library = 'library',
-  netflow = 'netflow',
-  plain = 'plain',
-  registry = 'registry',
-  suricata = 'suricata',
-  system = 'system',
-  system_dns = 'system_dns',
-  system_endgame_process = 'system_endgame_process',
-  system_file = 'system_file',
-  system_fim = 'system_fim',
-  system_security_event = 'system_security_event',
-  system_socket = 'system_socket',
-  threat_match = 'threat_match',
-  zeek = 'zeek',
-}
+export const RowRendererCount = Object.keys(RowRendererIdEnum).length;
+export const RowRendererValues = Object.values(RowRendererId.Values);
```

Comment:
`Object.values(RowRendererId.Values)` was copied in a bunch of places, so I created this shared export